scalafmt-tests/src/test/resources/rewrite/SortImports.stat

```diff
@@ -35,8 +44,8 @@ import a.{
   bar
 }
 >>>
-import a.{zzzz => _, bar}
-<<< caveat: comments
+import a.{bar, zzzz => _}
+<<< SKIP caveat: comments
```

Comment:
Had to skip this test. For some reason the comments was being removed, but I'm not sure why. 

Though the test is confusing since the comment stays in the same place but is now pointing to the wrong importee.

I don't see the point of having comments in imports but if this is really needed I can dig deeper into what is happening.

Reply:
Oh, I know why the comment is being removed: because I remove all tokens but `Importer.importees` does NOT return comments, so when we do the rewrite, the comment is gone. 

Unn, not sure how this was working before.